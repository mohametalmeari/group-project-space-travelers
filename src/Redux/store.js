import { configureStore } from '@reduxjs/toolkit';
import missionReducer from './features/missions/missionSlice';

const store = configureStore({
  reducer: {
    mission: missionReducer,
  },
});

export default store;
