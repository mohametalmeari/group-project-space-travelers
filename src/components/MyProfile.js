import ReservedRocketDisplay from '../RocketComponent/ReservedRockets';

const MyProfile = () => (
  <>
    <section>
      <h2>My Missions</h2>
      <ul>
        <li>Telstar</li>
      </ul>
    </section>
    <section>
      <ReservedRocketDisplay />
    </section>
  </>
);
export default MyProfile;
