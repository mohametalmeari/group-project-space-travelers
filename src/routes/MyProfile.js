const MyProfile = () => (
  <div>
    <p>MyProfile Comming soon!</p>
  </div>
);
export default MyProfile;
