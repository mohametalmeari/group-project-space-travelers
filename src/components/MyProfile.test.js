import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../Redux/store';
import MyProfile from './MyProfile';

describe('MyProfile Component', () => {
  test('Test render of MyProfile', () => {
    const { container } = render(
      <MemoryRouter>
        <Provider store={store}>
          <MyProfile />
        </Provider>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
