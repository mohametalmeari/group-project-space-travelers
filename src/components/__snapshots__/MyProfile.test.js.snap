// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile Component Test render of MyProfile 1`] = `
<div>
  <div
    class="my-profile"
  >
    <section>
      <div
        class="joinedMissionsContainer"
      >
        <h2
          class="joinedMissionsHeader"
        >
          My Missions
        </h2>
        <p
          class="noMissionsJoined"
        >
          No missions joined
        </p>
      </div>
    </section>
    <section>
      <div
        class="container"
      >
        <h3>
          My Rockets
        </h3>
        <div>
          <div
            class="colums"
          >
            <p>
              Currently no rockets reserved.
            </p>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
